adapters/f5_rest/f5_rest_command.php

```diff
@@ -84,9 +84,13 @@ function eval_IMPORT() {
 				}
 				foreach ( $parser_list as $op_eval => $sub_parsers ) {
 					
-					$running_conf = sendexpectone ( __FILE__ . ':' . __LINE__, $sms_sd_ctx, $op_eval, "" );
-					foreach ( $sub_parsers as $parser ) {
-						$parser->parse ( $running_conf, $objects );
+					try {
```

Comment:
There is already a try/catch in this function.
If you just want to add a log you should use sms_log_error() function. echo will output only if debug are enabled.
Something like that:
sms_log_error(__FILE__ . ':' . __LINE__ . ": " . $e->getMessage () . "\n" );

Reply:
Yes, you are right. But this exception leads to MS execution error. And MS execution fails (stops).
Exception handler implemented here in this PR allows MS execution being finished normally.